### PR TITLE
Prevent max from going negative, fix warning, and change footer link

### PR DESCRIPTION
**Pull request by Ely-S**

### CODE REVIEW

File: app/components/max.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ interface MaxProps {
   maxValueLabel: string;
 }
 const formatMaxValue = (v: string): string => {
-  return parseFloat(v).toFixed(2);
+  return Math.max(0, parseFloat(v)).toFixed(2);
```
Comment (Ely-S) [inline app/components/max.tsx:7]: Fixes this.

localhost_3000_

File: app/components/Footer.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ export default function Footer() {
   let productLinks: Link[] = [
     {
       label: "Docs",
-      url: "https://tender-fi.gitbook.io/tender.fi-docs/introduction/what-is-tender.fi",
+      url: "https://docs.tender.fi",
```
Comment (Ely-S) [inline app/components/Footer.tsx:9]: we moved our gitbook to subdomain

File: app/components/account-summary/ring.tsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+import { useEffect, useState } from "react";
 
 const NEAR_ZERO_PERCENT = 385;
 const NEAR_HUNDRED_PERCENT = 760;
 
 export default function Ring({ percent }: { percent: number }) {
   let [strokeOffset, setStrokeOffset] = useState<number>(NEAR_ZERO_PERCENT);
 
-  useLayoutEffect(() => {
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (Ely-S) [reply app/components/account-summary/ring.tsx:9]: This hook doesn't need to read or manipulate the DOM, so `useLayoutEffect` is not necessary and `useEffect` can be used.

[the remark this answers is not in the document]
Comment (mitchtrale) [reply app/components/account-summary/ring.tsx:9]: good stuff

**Review by mitchtrale — APPROVED** [review]

**Pull request merged**